source/Octopurls/redirects.json

```diff
@@ -707,5 +707,6 @@
   "hybridlivestream": "https://www.techielass.com/the-future-of-hybrid/",
   "AWSAuthenticationCredentials": "https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-credentials-node.html",
   "AWSAuthenticationIAMRoles": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use.html",
-  "AWSAuthenticationExternalId": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html"
+  "AWSAuthenticationExternalId": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html",
+  "ServiceNowIntegration": "https://docs.google.com/document/d/16OKPBZ4EFtAPtpXlCznT-4DR2bI0TwicEMStM7zPzCM/edit?usp=sharing"
```

Comment:
This document link is not publically accessible so the redirect will fail testing. Or at least that used to be the case, there is a test somewhere that ensures that all redirects get a 200.
I check in the `#feed-octopurls` channel https://octopusdeploy.slack.com/archives/C625CE6R3/p1652397082086909